src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -0,0 +1,349 @@
+        />
+      </div>
+    )
+
+    return (
+      <div className={styles.blockNavbar}>
+        <div className={styles.navbar}>
+          <Layout.ScrollDetector>
+            <Hamburger />
+            <div
+              className={
+                showMobileMenu
+                  ? [styles.phoneAndTablet, styles.visible].join(' ')
+                  : styles.phoneAndTablet
+              }
+            >
+              <div
+                className={
+                  showMobileMenu ? styles.mobileMenu : styles.hideMobileMenu
+                }
+              >
+                <NavLink
+                  href={logoHref}
+                  LinkComponent={LinkComponent}
+                  className={styles.phoneLogo}
+                >
+                  {LogoWhite({ className: styles.logo })}
+                </NavLink>
+                <Hamburger />
+                <div className={styles.accordion}>
+                  {LINKS.NAVLINKS.map((link, idx) => (
+                    <div
+                      key={link.id}
+                      className={
+                        idx === activeAccordionItem
+                          ? [styles.accordionItem, styles.active].join(' ')
+                          : styles.accordionItem
+                      }
+                      onClick={() => this.toggleAccordionItem(idx)}
+                      onKeyPress={(e) =>
+                        this.handleAccordionItemKeyPress(e, idx)
+                      }
+                      tabIndex={0}
+                      role="button"
+                    >
+                      <div className={styles.accordionParent}>
+                        <TitleMedium.Sans.Regular400>
+                          {link.title}
+                        </TitleMedium.Sans.Regular400>
+                        <AccordionToggleIcon />
+                      </div>
+                      <div className={styles.accordionChildren}>
+                        {link.subnav &&
+                          link.subnav.items.length > 0 &&
+                          link.subnav.items.map((link) => (
+                            <div className={styles.accordionChild}>
+                              <TitleSmall.Sans.Light300>
+                                <NavLink
+                                  href={link.href}
+                                  LinkComponent={
+                                    link.href !== '/login/'
+                                      ? LinkComponent
+                                      : null
+                                  }
+                                >
+                                  {link.title}
+                                </NavLink>
+                              </TitleSmall.Sans.Light300>
+                            </div>
+                          ))}
+                      </div>
+                    </div>
+                  ))}
+                </div>
+                <div class={styles.belowAccordion}>
```

Comment:
Non-blocking: Can we please split out into a separate Component

Reply:
yes

Reply:
marked in TODOs for fastfollow